scripts/india_udise/common/base_school_data.py

```diff
@@ -104,6 +104,45 @@ class GeographicLevel(enum.Enum):
     BLOCK = "block"
 
 
+ATTRIBUTE_MAPPING = {
+    "cat1": {
+        "levelOfSchool": "PrimarySchool_Grade1To5"
+    },
+    "cat2": {
+        "levelOfSchool": "UpperPrimarySchool_Grade1To8"
+    },
+    "cat3": {
+        "levelOfSchool": "HigherSecondarySchool_Grade1To12"
+    },
+    "cat4": {
+        "levelOfSchool": "UpperPrimarySchool_Grade6To8"
+    },
+    "cat5": {
+        "levelOfSchool": "UpperPrimarySchool_Grade6To12"
+    },
+    "cat6": {
+        "levelOfSchool": "SeniorSecondarySchool_Grade1To10"
+    },
+    "cat7": {
+        "levelOfSchool": "SeniorSecondarySchool_Grade6To10"
+    },
+    "cat8": {
+        "levelOfSchool": "SeniorSecondarySchool_Grade9To10"
+    },
```

Comment:
if 'cat9' is expected to be dropped, pls add a  comment.

Reply:
Done. 

![udise_school_category](https://user-images.githubusercontent.com/76114/142035697-6e6e32ba-f34b-43bc-9df6-516c8a6557cb.png)